### ISSUE TITLE
parameter settings different with paper reported.

### ISSUE THREAD

**Issue by snownus** [kakaobrain/fast-autoaugment#32]
@ildoonet , I refer to the paper. 

1. At the search stage, #epochs =90. while in your code is 270. If we set the #epochs=90, how to set the lr policy?
3. At the search stage, #training samples=6000. while in your code, for reduced_imagenet, the #training samples seems almost 50000. 

Can you explain this? And which one should I follow?

Many thanks.

**Comment by ildoonet** [comment]
@snownus Thanks for the report.

1. You need to provide `--epoch 90` argument, when you run the search.py file. Learning rate scheduler will be adapted accordingly: https://github.com/kakaobrain/fast-autoaugment/blob/master/FastAutoAugment/lr_scheduler.py#L12

2. I will fix the parameter soon. The paper's parameters should be used. Since we conducted ablation studies internally after submitting the paper, some parameters are incorrect but it is not intentional. 

Thanks.

**Comment by snownus** [comment]
@ildoonet thanks for your instant reply.
